charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,107 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
```

Comment:
_to Working_
_to Assisting_
_to Helping_

doesn't scan. Maybe simplify the second sentence to:

 _This team is:_
-_Working_
-_Assisting_